web_poet/testing/utils.py

```diff
@@ -0,0 +1,44 @@
+import json
+import os
+from pathlib import Path
+from typing import Any, Iterable, Optional, Union
+
+from itemadapter import ItemAdapter
+
+from web_poet.serialization import SerializedDataFileStorage, serialize
+
+INPUT_DIR_NAME = "inputs"
+OUTPUT_FILE_NAME = "output.json"
+META_FILE_NAME = "meta.json"
+
+
+def _get_available_filename(template: str, directory: Union[str, os.PathLike]) -> str:
+    i = 1
+    while True:
+        result = Path(directory, template.format(i))
+        if not result.exists():
+            return result.name
+        i += 1
```

Comment:
I slighly prefer to continue from the last number, without trying to fill the gaps, but either way is fine.